src/main.cpp

```diff
@@ -542,19 +582,26 @@ int main(int argc, char**argv)
   seed = time(NULL) ^ getpid();
   srand(seed);
 
+  // Create a DNS resolver.
+  int af = AF_INET;
+  struct in6_addr dummy_addr;
+  if (inet_pton(AF_INET6, options.local_host.c_str(), &dummy_addr) == 1)
+  {
+    TRC_DEBUG("Local host is an IPv6 address");
+    af = AF_INET6;
+  }
+
+  DnsCachedResolver* dns_resolver = new DnsCachedResolver("127.0.0.1");
```

Comment:
I think you want to honour a --dns-servers option, like in https://github.com/Metaswitch/sprout/blob/b324dcfec4f3b489bfad8bb78101774ee2ea281d/src/main.cpp#L1704. (127.0.0.1 is dnsmasq, which we only have in the signaling namespace, not in management - if this code comes from somewhere else we may want to fix the other place up too.)